Kickstarter-iOS/Views/Cells/CreditCardCell.swift

```diff
@@ -0,0 +1,53 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
```

Comment:
Looks like this import is not necessary. Maybe we can remove it in the future (should not block the release)